evm_test_runner/src/test_dir_reading.rs

```diff
@@ -37,6 +37,23 @@ pub(crate) struct Test {
     pub(crate) info: ParsedTest,
 }
 
+pub(crate) fn get_default_parsed_tests_path() -> anyhow::Result<PathBuf> {
+    std::env::current_dir()?
+        .ancestors()
+        .map(|ancestor| {
+            let mut buf = ancestor.to_path_buf();
+            buf.push(GENERATION_INPUTS_OUTPUT_DIR);
+            buf
+        })
+        .find(|path| path.exists())
+        .ok_or_else(|| {
+            anyhow!(
+                "Unable to find {} in cwd ancestry.",
+                GENERATION_INPUTS_OUTPUT_DIR
+            )
+        })
+}
+
```

Comment:
Just thinking about this some more. If we're running the runner, then we're likely going to either be in `evm_tests` or `evm_tests/evm_test_runner`. I think in both of these cases it's not going to find `generation_inputs`. Maybe instead we should be looking for `evm_tests` and then attempting to reconstruct the path?

Reply:
Hmm... I think if we are distributing both binaries under the root `evm_tests`, I still feel like both are possible, even outside of a development setting.

I'm good with either here. Also fine with what you currently have. You call it!